datasync/chaindatafetcher/kafka/consumer.go

```diff
@@ -306,37 +307,99 @@ func (c *Consumer) updateOffset(buffer [][]*Segment, lastMsg *sarama.ConsumerMes
 	return nil
 }
 
+// renewExpireMsg updates expiration timer and message if the expiring message is handled.
```

Comment:
It does not update the message, message is updated by the returned value of this function at caller-side.

Reply:
Right. The timer is reset only, so I renamed it to `resetTimer`. PTAL.